### PR TITLE
Add missing dashboard fields

### PR DESCRIPTION
**Pull request by mnaboka**
Issue #185

If applied this Pull request will add some of the missing fields to `DashboardLite` and adds a generated `datadog-accessors.go` file.
Specifically:
 - `ReadOnly` type `bool`
 - `Created` type `string`
 - `Modified` type `string`
 - `CreatedBy` type `CreatedBy`

A custom type `CreatedBy` has the following fields:
 - `Disabled` type `bool`
 - `Handle` type `string`
 - `Name` type `string`
 - `IsAdmin` type `bool`
 - `Role` type `string`
 - `AccessRole` type string
 - `Verified` type `bool`
 - `Email` type `string`
 - `Icon` type `string`

PR also includes a unittest.

### CODE REVIEW

File: dashboards_test.go [hunk 1]
```diff
@@ -75,3 +78,109 @@ func (suite *YAxisTestSuite) TestYAxisJSONUnmarshalAutoMinMax() {
...
+	}
+
+	expectedReadOnly := false
+	if readOnly, ok := d1.GetReadOnlyOk(); !ok || readOnly != expectedReadOnly {
+		t.Fatalf("expect read_only exist and be set to %v. Got exists %v, value %v", expectedReadOnly, ok, readOnly)
+	}
+
+	expectedModified := "2018-09-11T06:38:09.000000+00:00"
+	if modified, ok := d1.GetModifiedOk(); !ok || modified != expectedModified {
+		t.Fatalf("expect modified field to exist and be set to %s. Got exists %v, value %s", expectedModified, ok, modified)
```
Comment (ojongerius) [inline dashboards_test.go]: I see this with the other checks too; if (in this case) `Modified` was *not* set, `GetModifiedOk` will return an empty string and boolean false. In that case this check will fail with (notice value is empty): 

> expect modified field to exist and be set to 2018-09-11T06:38:09.000000+00:00. Got exists false, value

File: tests/fixtures/dashboards_response.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{
```
Comment (ojongerius) [inline tests/fixtures/dashboards_response.json]: 🥇

**Comment by ojongerius** [comment]
Thanks for the contribution @mnaboka!

File: dashboards_test.go (hunk 1 above)
Comment (mnaboka) [reply dashboards_test.go]: fixed!

**Pull request merged**